### PR TITLE
OSOE-49: Enforce Windows newlines for C# files to avoid false positives with IDE0055 warning.

### PR DESCRIPTION
**Pull request by sarahelsaig**
[OSOE-49](https://lombiq.atlassian.net/browse/OSOE-49)

### CODE REVIEW

**Review by Piedone — APPROVED** [review]

**Review by Piedone — CHANGES_REQUESTED** [review]
Just noticed due to a failing test that a Theme.png is needed in both the BaseTheme and the Sample.

**Review by Piedone — APPROVED** [review]

File: Lombiq.BaseTheme.Samples/Services/ResourceFilters.cs [hunk 1]
```diff
@@ -4,7 +4,9 @@
 namespace Lombiq.DataTables.Samples.Navigation;
 
 // This service implements conditional configuration to direct the ResourceFilterMiddleware when and what styles need to
-// be added to the resources.
+// be added to the resources. The attribute indicates that the middleware should only invoke this provider if the
+// current admin or site theme is the provided.
```
Comment (0liver) [inline Lombiq.BaseTheme.Samples/Services/ResourceFilters.cs]:
```suggestion
// current admin or site theme is equal to the one in the attribute constructor.
```

File: Lombiq.BaseTheme/Services/ResourceFilters.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+using Lombiq.HelpfulLibraries.OrchardCore.ResourceManagement;
+
+namespace Lombiq.DataTables.Navigation;
+
+[ResourceFilterThemeRequirement(FeatureIds.BaseTheme)]
+public class ResourceFilters : IResourceFilterProvider
+{
+    public void AddResourceFilter(ResourceFilterBuilder builder) =>
+        builder.Always().RegisterStylesheet(ResourceNames.Site);
+}
```
Comment (0liver) [inline Lombiq.BaseTheme/Services/ResourceFilters.cs:11]: How does using this filter stack up against the comment [here](https://github.com/Lombiq/Orchard-Base-Theme/pull/16/files#diff-137889361196787b5e675fd3837fb3a3d2786726567ddd5ee771b029aa1a51f2R14)?
> If you only have "Always" styles then it's easier to inject them via the Views/Widget-LayoutInjection.cshtml file and skip using a ResourceFilterMiddleware.

**Review by 0liver — CHANGES_REQUESTED** [review]

File: Lombiq.BaseTheme/Services/ResourceFilters.cs (hunk 2 above)
Comment (sarahelsaig) [reply Lombiq.BaseTheme/Services/ResourceFilters.cs:11]: I don't fully agree with that comment any more (past self is always a dumdum 😄) and I have reworded it:
> We always want to add the site style. If you only have "Always" styles and you are going to override Views/Widget-LayoutInjection.cshtml anyway, then it's easier to inject your style there and skip using a ResourceFilterMiddleware altogether.

In the base theme we have no other use for a _Widget-LayoutInjection.cshtml_ override so just adding this filter is simpler.

Comment (0liver) [reply Lombiq.BaseTheme/Services/ResourceFilters.cs:11]: That makes sense. Sounds good like that.

**Review by 0liver — APPROVED** [review]

**Pull request merged**